### PR TITLE
Update `index-of` and `slice` expression Android 11.0.0

### LINKED ISSUE #159: MapLibre Native support for index-of
After https://github.com/maplibre/maplibre-native/pull/1113 is merged & released MapLibre Native (Android) will have support for `index-of`.

https://maplibre.org/maplibre-style-spec/expressions/#index-of

Creating an issue now so I don't forget later.

### PR DESCRIPTION
**Pull request by louwers**
Will remove Draft status of this PR once 11.0.0 has been released.

Resolves #159

### CODE REVIEW

**Review by HarelM — APPROVED** [review]

**Comment by HarelM** [comment]
@louwers any updates on this? Can this be merged?

**Comment by louwers** [comment]
@HarelM No it is not released yet.

**Comment by acalcutt** [comment]
Just an FYI, I just made a PR that adds 'slice' and 'index-of' to node also
https://github.com/maplibre/maplibre-native/pull/2047

If this gets merged does the node version need to be added similar to this PR?

**Comment by louwers** [comment]
@acalcutt We are not currently tracking style support of Node.js. As you can see:

https://maplibre.org/maplibre-style-spec/expressions/

If you want to, you can add it, but you'd have to go over all the other things as well!

**Comment by HarelM** [comment]
@louwers this is a reminder since I saw you are creating apre release of 11.

**Comment by louwers** [comment]
There's no official release out with these features.

**Comment by HarelM** [comment]
@louwers didn't you release version 11?

File: src/reference/v8.json [hunk 1]
```diff
@@ -2822,6 +2822,7 @@
         "sdk-support": {
           "basic functionality": {
             "js": "1.10.0",
+            "android": "11.0.0",
```
Comment (louwers) [inline src/reference/v8.json:2825]:
```suggestion
            "android": "10.3.0",
```

File: src/reference/v8.json [hunk 2]
```diff
@@ -2839,6 +2840,7 @@
...
```
Comment (louwers) [inline src/reference/v8.json:2843]:
```suggestion
            "android": "10.3.0",
```

**Review by HarelM — APPROVED** [review]

**Pull request merged**